aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandlerTest.java

```diff
@@ -25,14 +27,16 @@
 import software.amazon.cloudformation.proxy.ProgressEvent;
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 
+import java.util.Arrays;
+
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.lenient;
 import static org.mockito.Mockito.mock;
-import static software.amazon.codeguruprofiler.profilinggroup.RequestBuilder.makeInvalidRequest;
-import static software.amazon.codeguruprofiler.profilinggroup.RequestBuilder.makeValidRequest;
+import static software.amazon.codeguruprofiler.profilinggroup.RequestBuilder.*;
```

Comment:
Could we leave them expanded please? I think that is the code style we have been using so far.

Reply:
Yes, I will update them to be expanded, thanks.